docs/usage/configuration/config_documentation.md

```diff
@@ -567,6 +567,19 @@ Example configuration:
 dummy_events_threshold: 5
 ```
 ---
+Config option `delete_stale_devices_after`
+
+An optional duration. If set, Synapse will run an hourly background task to log out and
```

Comment:
I do not know what is the best time, but every hour seems to be very often. Is one run per day not enough?

Reply:
Good point, one per day should be more than enough.